Knoema.Client/Client.cs

```diff
@@ -63,8 +72,8 @@ public Client(string host, string token)
 			_token = token;
 		}
 
-		public Client(string host, string clientId, string clientSecret)
-			: this()
+		public Client(string host, string clientId, string clientSecret, bool ignoreCertErrors = false, string scheme = "")
```

Comment:
Why cannot you just add one ctor with Ignorecert and scheme instead of changing all of the ctors. and still question above is valid

Reply:
How do you pass host, appId, appSecrec and token parameters then?

Reply:
I mean one more ctor like this Client(host, clientId, clientSecret, ignoreCertErrors, scheme)